### ISSUE TITLE
TVB compatible datasets

### ISSUE THREAD

**Issue by sipv** [the-virtual-brain/tvb-recon#37]
Current bash scripts based pipeline (https://github.com/the-virtual-brain/tvb-recon/blob/94d7cd25aaf257a70e965d85adaa278901512511/bin/main.sh) does not produce TVB compatible structural datasets (i.e. files areas.txt, average_orientations.txt, weights.txt, tract_lengths.txt, cortical.txt, centres.txt), but stops with the connectome.

Some code meant to generate this TVB dataset is in the old `scripts` pipeline, but it is somehow out-of-date (considering the versions of the software in the virtual machine). Furthermore, it is quite complex: it depends on at least six python scripts, remesher in C and some region mapping code in MATLAB.

I would need this functionality ASAP, but before I start playing with this (either dusting off the original python scripts, or rewriting them to the new python library) can you summarize the status of the related work in the new python version of the pipeline? @maedoc @dionperd 

For example, some of the code related to remeshing (https://github.com/the-virtual-brain/tvb-recon/blob/94d7cd25aaf257a70e965d85adaa278901512511/tvb/recon/algo/head_sensors.py) and subparcellation (https://github.com/the-virtual-brain/tvb-recon/blob/94d7cd25aaf257a70e965d85adaa278901512511/tvb/recon/algo/service/subparcellation.py) is ported to the python library. Does it work, and is it a final version?

**Comment by maedoc** [comment]
> it depends on at least six python scripts, remesher in C and some region mapping code in MATLAB.

Only the Python scripts should be necessary.  Remesher is replaced by the Bash script `resample_anat` or similar. 

The subparcellation can be done with the existing tools in this library, yes.

**Comment by sipv** [comment]
OK, that is good.

And for the region mapping there is `tvb.recon.io.AnnotationIO`, right? Though I still have to take care of relabeling regions, if I haven't missed anything.
